scania-digital-design-system/sdds
# fix(message): cleaned up and restructured the story for the message component

**Describe pull-request**  
Cleaned up and restructured the story for the message component. Added the possibility to switch between native/wecomponent or no icon. 

**Solving issue**  
Fixes: [AB#2521](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2521)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in Components -> Message
3. Check that the controls work as intended and that the design reflects the on in Figma. 

**Screenshots**  
-

**Additional context**  
-
